### PR TITLE
Responsive CSS Image Slider 

### PR DESCRIPTION
**Pull request by zaidansari42**
Please check and let me know if I have done any mistakes.

### CODE REVIEW

**Review by ManSOorcode — APPROVED** [review]

**Review by mujeeb100 — APPROVED** [review]
Nice

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+}
+
+.container .slider .slide .caption p {
+  font-size: 18px;
+  transition: 0.5s;
+}
+
+.container .slider .slide.slide1 {
+  background-image: url(https://www.motortrend.com/uploads/sites/5/2020/08/2021-Nissan-GT-R-NISMO-front-three-quarter-motion.jpg?fit=around%7C1000:625);
+  background-size: cover;
```
Comment (Rehan-Khan-Engineer) [inline styles.css:55]: Using links to add background images is not a good idea as the links might be removed in the future. For now its alright

**Review by Rehan-Khan-Engineer — APPROVED** [review]
LGTM

**Review by murtazshaikh — APPROVED** [review]
Awesome Work

File: styles.css (hunk 1 above)
Comment (zaidansari42) [reply styles.css:55]: Okay, Will always use local media next time onwards

**Review by syedabdullahali — APPROVED** [review]

**Pull request merged**